lib/processAssets.js

```diff
@@ -1,33 +1,31 @@
-const { flattenName, prepareAssetBundleForPDF} = require('./processShared')
+const { flattenName, prepareAssetBundleForPDF, prepareAssetBundleForSVG} = require('./processShared')
 const svg = require('sketch-to-svg')
 const fs = require('fs')
 const pdfDoc = require('pdfkit')
 const SVGtoPDF = require('svg-to-pdfkit')
 const path = require('path')
 
-async function processAssets(layers, filepath, outputFolder) {
-  let css = []
+async function processAssets(layers, filepath, outputFolder, outputAsSVG) {
+  let assets = []
 
-  let assetBundleFolder = createAssetBundle(outputFolder)
+  const assetBundleFolder = createAssetBundle(outputFolder)
 
   for (let index = 0; index < layers.length; ++index) {
-    let layer = layers[index]
-    let backgroundImage = await assetToSvg(layer, filepath, outputFolder, flattenName(layer.name))
-    let filePath = prepareAssetBundleForPDF(assetBundleFolder, flattenName(layer.name))
-    svgToPdf(backgroundImage, filePath, {width: assetWidth(layer), height: assetHeight(layer)})
-    css.push({
+    const layer = layers[index]
+    const backgroundImage = await assetToSvg(layer, filepath, outputFolder, flattenName(layer.name))
+    if (outputAsSVG) {
+      const assetPath = prepareAssetBundleForSVG(assetBundleFolder, flattenName(layer.name))
+      fs.writeFileSync(assetPath, backgroundImage)
+    } else {
+      const assetPath = prepareAssetBundleForPDF(assetBundleFolder, flattenName(layer.name))
+      svgToPdf(backgroundImage, assetPath, {width: assetWidth(layer), height: assetHeight(layer)})
+    }
+    assets.push({
       name: flattenName(layer.name),
-      css: [
-        { name: "background-image", value: backgroundImage },
-        { name: "background-repeat", value: "no-repeat" },
-        { name: "background-position", value: "center" },
-        { name: "width", value: assetWidth(layer) },
-        { name: "height", value: assetHeight(layer) }
-      ]
```

Comment:
Why has this css been removed?

Reply:
I suspect it was there from a copy-paste job from the sketch-to-css project. It's not used here so I removed it.